### ISSUE TITLE
Fix typo in TCPServerTests test case name

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6821]
|<img alt="atorkhov's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| atorkhov reported|
|-|-|
|Trac ID|trac#6821|
|Type|enhancement|
|Created|2013-11-10 07:03:39Z|

Fix typo in TCPServerTests test case name. I think that
test_writeAfteDisconnectAfterTLS and test_writeSequenceAfteDisconnectAfterTLS test cases should be called as test_writeAfterDisconnectAfterTLS test_writeSequenceAfterDisconnectAfterTLS. Changing doesn't break anything. Patch attached.

Attachments:

* [fix-typo-in-TCPServerTests-test-case-name.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/b79/b7946664a2d40d9f93486d8e9ed87216c1448277/26fec86468ef37691942a8afbd42ab38aae04122.patch) (899 bytes) - added by atorkhov on 2013-11-10 07:04:58Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__6821 6821
type__enhancement enhancement
reporter__atorkhov atorkhov
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1384067019000000 1384067019000000
changetime__1384091392000000 1384091392000000
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set owner to @exarkun<br>@exarkun set status to `assigned`|
|-|-|

Thanks.  This change looks great to me.  I'll apply it to trunk.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set status to `closed`|
|-|-|

(In [40627]) Apply fix-typo-in-TCPServerTests-test-case-name.patch

Author: atorkhov
Reviewer: exarkun
Fixes: [#6821](https://github.com/twisted/twisted/issues/6821)

Fix a typo in the names of two test methods in `twisted.internet.test.test_tcp`.
